lib/govuk_app_config/railtie.rb

```diff
@@ -8,6 +8,10 @@ class Railtie < Rails::Railtie
       end
     end
 
+    initializer "govuk_app_config.configure_open_telemetry" do |app|
+      GovukOpenTelemetry.configure(app.class.module_parent_name.underscore)
```

Comment:
You've probably already thought of this, but you'll need to make the service name configurable at some point if this gets used for real, otherwise things like the draft- deployments will be hard to distinguish from the regular ones etc. No problem for now though.

Reply:
Yup good point! Also this won't work for our non-rails apps e.g. search api